webrender/src/device/gfx.rs

```diff
@@ -1636,37 +1643,37 @@ impl<B: hal::Backend> DescriptorPools<B> {
         let cache_clip_range = vec![
             hal::pso::DescriptorRangeDesc {
                 ty: hal::pso::DescriptorType::SampledImage,
-                count: 400,
+                count: descriptor_count * 10,
```

Comment:
where does 10 come from?

Reply:
This 10 comes from the shader, e.g. in a cache_clip shader we have 1 uniform buffer, 10 sampler and 10 image. I will check these numbers again, and move them in a `const` value.